packages/react-composites/tests/browser/chat/FileSharing.test.ts

```diff
@@ -194,3 +200,54 @@ test.describe('Filesharing SendBox Errorbar', async () => {
     expect(await page.screenshot()).toMatchSnapshot('filesharing-sendbox-file-upload-in-progress-error.png');
   });
 });
+
+test.describe('Filesharing Message Thread', async () => {
+  test.skip(isTestProfileStableFlavor());
+
+  test.beforeEach(async ({ pages, users, serverUrl }) => {
+    const usersWithArgs: { user: ChatUserType; qArgs?: { [key: string]: string } }[] = [];
+    for (let idx = 0; idx < users.length; idx++) {
```

Comment:
This can be simplified to:

```
const firstUserArgs = { user: users[0] qargs: { /* qargs for the first user */ }};
const otherUsersArgs = users.slice(1).map(user => ({user, qArgs: { /* qargs for other users */ }));
const usersArgs = [firstUserArgs, ...otherUsersArgs];